application/views/questionAdministration/subquestionRow.twig

```diff
@@ -129,18 +129,21 @@
         </div>
     </td>
 
+    {% set relevanceTooltip = supportsRelevance ? gT("Click to expand") : gT("Relevance is not currently supported for columns") %}
+
     <!-- Relevance equation -->
     {% if first and scale_id == 0 %}
         <td class="relevance-equation">
```

Comment:
If must be shown something like this : <td class="relevance-equation{% if !supportsRelevance %} hidden-td{% endif %}">
Or another class, to set of column to 0